Tests/SourceKittenFrameworkTests/SourceKitTests.swift

```diff
@@ -12,13 +12,30 @@ import XCTest
 
 private func run(executable: String, arguments: [String]) -> String? {
     let task = Process()
-    task.launchPath = executable
     task.arguments = arguments
 
     let pipe = Pipe()
     task.standardOutput = pipe
 
-    task.launch()
+    do {
+    #if canImport(Darwin)
```

Comment:
This pattern is repeated enough times to be worth a small helper. If it's internal to SourceKittenFramework we can still use it from tests by using `@testable import SourceKittenFramework` without polluting our public API.

Reply:
Certainly it would be better to refoctoring. But is it better to include them in this PR?

Reply:
No it's ok, we can do it later.

Reply:
#586